Doc/library/stdtypes.rst

```diff
@@ -2276,15 +2276,18 @@ expression support in the :mod:`re` module).
    alternatives provides their own trade-offs and benefits of simplicity,
    flexibility, and/or extensibility.
 
-String objects have one unique built-in operation: the ``%`` operator (modulo).
-This is also known as the string *formatting* or *interpolation* operator.
-Given ``format % values`` (where *format* is a string), ``%`` conversion
-specifications in *format* are replaced with zero or more elements of *values*.
-The effect is similar to using the :c:func:`sprintf` in the C language.
+The % operator (modulo) can also be used for string formatting. Given ``'string'
+% values``, instances of ``%`` in ``string`` are replaced with zero or more
+elements of ``values``. This operation is commonly known as string
```

Comment:
I'd have to agree with Jelle's comment and also note that what's common can be somewhat subjective (for example, I'd search with the query `string formatting` instead of `string interpolation`). Would you be interested in updating the PR to address the review?